### PR TITLE
ejson: 1.3.3 -> 1.5.2

### PR DESCRIPTION
**Pull request by joshheinrichs-shopify**
## Description of changes

I'm currently seeing errors when trying to use ejson on Darwin:

```console
$ nix-shell -p ejson
...
AbstractMemory.c:586:9: error: call to undeclared function 'rb_safe_level'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
    if (rb_safe_level() >= 1 && OBJ_TAINTED(str)) {
        ^
...
```

In 1.5.2, man pages were removed[^1], so Ruby isn't required anymore.

[^1]: https://github.com/Shopify/ejson/pull/152

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by mattpolzin** [comment]
Result of `nixpkgs-review pr 341139` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ejson</li>
  </ul>
</details>

File: pkgs/development/tools/ejson/default.nix [hunk 1]
```diff
@@ -1,39 +1,20 @@
 { lib, bundlerEnv, ruby, buildGoModule, fetchFromGitHub }:
```
Comment (mattpolzin) [inline pkgs/development/tools/ejson/default.nix:1]:
```suggestion
{ lib, buildGoModule, fetchFromGitHub }:
```

**Review by mattpolzin — CHANGES_REQUESTED** [review]
You've got a couple of extra arguments at the top now that you've removed the need for them. While you're at it, may as well `nix run .#nixfmt-rfc-style -- -s pkgs/development/tools/ejson/default.nix` since everything is going to get formatted sooner or later.

File: pkgs/development/tools/ejson/default.nix (hunk 1 above)
Comment (joshheinrichs-shopify) [reply pkgs/development/tools/ejson/default.nix]: oops good catch. fixed and formatted!

**Comment by mattpolzin** [comment]
Result of `nixpkgs-review pr 341139` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ejson</li>
  </ul>
</details>

**Comment by mattpolzin** [comment]
Result of `nixpkgs-review pr 341139` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ejson</li>
  </ul>
</details>

**Review by mattpolzin — APPROVED** [review]

**Pull request merged**